plugins/discovery-ec2/src/main/java/org/elasticsearch/discovery/ec2/AwsEc2ServiceImpl.java

```diff
@@ -79,7 +79,7 @@ static ClientConfiguration buildConfiguration(Logger logger, Ec2ClientSettings c
         // Increase the number of retries in case of 5xx API responses
         final Random rand = Randomness.get();
         final RetryPolicy retryPolicy = new RetryPolicy(
```

Comment:
Can we avoid setting our own retry policy at all and use the SDK default? The SDK makes a more nuanced decision about how much to back off based on the error type. If we really want 10 retries then that can be overridden with `setMaxErrorRetry(10)` instead?

Reply:
Right, that's much nicer. Moved to the default and kept retries at `10` for now in https://github.com/elastic/elasticsearch/pull/50550/commits/d015576900c80123733e36ce2a8f5a4b8c897b41 :)